python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -186,18 +209,30 @@ def getExposureId(self, dataRef):
         return int(dataRef.get("ccdExposureId", immediate=True))
 
     def fetchReferences(self, dataRef, exposure):
-        """Return a SourceCatalog of sources which overlap the exposure.
-
-        The returned catalog is sorted by ID and guarantees that all included children have their
-        parent included and that all Footprints are valid.
-
-        @param dataRef       Data reference from butler corresponding to the image to be measured;
-                             should have tract, patch, and filter keys.
-        @param exposure      lsst.afw.image.Exposure to be measured (used only to obtain a Wcs and
-                             bounding box).
-
-        All work is delegated to the references subtask; see CoaddSrcReferencesTask for information
-        about the default behavior.
+        """Get sources that overlap the exposure.
+
+        Parameters
+        ----------
+        dataRef : `lsst.daf.persistence.ButlerDataRef`
+            Butler data reference corresponding to the image to be measured;
+            should have ``tract``, ``patch``, and ``filter`` keys.
+        exposure : `lsst.afw.image.Exposure`
+            The image to be measured (used only to obtain a WCS and bounding
+            box).
+
+        Returns
+        -------
+        lsst.afw.table.SourceCatalog
```

Comment:
Please name the return value everywhere. You can always use `result` if nothing else springs to mind.